Resources/Prototypes/ai_factions.yml

```
@@ -193,4 +206,4 @@
 - type: npcFaction
   id: Monkey
   hostile:
-  - Balloon     # Custom faction specifically for attacking bloons
\ No newline at end of file
+  - Balloon     # Custom faction specifically for attacking bloons
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)